### PR TITLE
[RTMD-145] - New page /confirmation

### PR DESCRIPTION
**Pull request by sulthan-ahmed**

### CODE REVIEW

File: apps/rotm/views/confirmation.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{{<partials-page}}
+  {{$page-content}}
+    {{>partials-confirmation-alert}}
+    {{#markdown}}confirmation{{/markdown}}
+  {{/page-content}}
+{{/partials-page}}
```
Comment (lennym) [inline apps/rotm/views/confirmation.html]: What's the logic behind extending this template and not using the default? https://github.com/UKHomeOfficeForms/hof-template-partials/blob/master/views/confirmation.html

File: apps/rotm/views/content/confirmation.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+##What happens next
+
+We have sent your anonymous report to the police team dealing with online terrorist material.
+
+[What did you think of this service? (takes 30 seconds)](https://eforms.homeoffice.gov.uk/outreach/Feedback.ofml?FormName=terrorism_reporting/)
+
+[Start a new report](/source)
+
+Attach one or more files to this issue
```
Comment (lennym) [inline apps/rotm/views/content/confirmation.md]: WUT?!

Comment (sulthan-ahmed) [reply apps/rotm/views/content/confirmation.md]: I'll delete that

File: apps/rotm/views/confirmation.html (hunk 1 above)
Comment (sulthan-ahmed) [reply apps/rotm/views/confirmation.html]: err that was initially a mistake

but I've now deleted the confirmation.html file and it won't show my markdown

Comment (sulthan-ahmed) [reply apps/rotm/views/confirmation.html]: I think I need to call my markdown what-happens-next

File: apps/rotm/views/partials/confirmation-alert.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<div class="alert-complete" role="alert">
+  <div class="alert-message">
+    <h1>{{#t}}pages.confirmation.alert-1{{/t}}
+    <div>{{#t}}pages.confirmation.alert-2{{/t}}</div></h1>
```
Comment (lennym) [inline apps/rotm/views/partials/confirmation-alert.html]: What's the deal with the random div? What's it there for?

Comment (sulthan-ahmed) [reply apps/rotm/views/partials/confirmation-alert.html]: it allows for the second part of the content to be on a new line as per the ticket

*No Div*
screen shot 2017-11-29 at 14 00 17

*With Div*
screen shot 2017-11-29 at 14 00 35

**Pull request merged**